CHANGES.md

```diff
@@ -8,6 +8,7 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Updated to [Metronome 0.6.33](https://github.com/dcos/metronome/tree/b8a73dd)
 * Updated to [Marathon 1.8.207](https://github.com/mesosphere/marathon/tree/9f3550487).
+* Updated to [Mesos 1.8.x](https://github.com/apache/mesos/tree/07d053f68b75505a4386913f05d521fa5e36373d)
```

Comment:
Can we update the tooling to make sure that there is an empty newline above and below this line?

And can we please link to the CHANGELOG file in the Mesos repository?

CC @bbannier 

The algorithm for now should be (copied from Slack):
```
Proposed algorithm for your bump bot for a _simple_ Mesos bump *between DC/OS patch releases*:

- modify `CHANGES.md` in the same branch
- look for _topmost_ "Notable changes" section
- detect: is line of format `* Updated to [Mesos <x.x.x-y>](https://github.com/apache/mesos/blob/<commit>/CHANGELOG).` already there?
- yes: update commit hash `<commit>` and ideally also version info `<x.x.x-y>`
- no: introduce line (with empty lines above and below)
```

Reply:
CC @orsenthil